### PR TITLE
Update hp_power_manager_login.rb

### PR DESCRIPTION
**Pull request by adamziaja**

### CODE REVIEW

File: modules/exploits/windows/http/hp_power_manager_login.rb [hunk 1]
```diff
@@ -42,7 +42,10 @@ def initialize(info = {})
       'Platform'       => 'win',
       'Targets'        =>
         [
-          [ 'Windows 2000 SP4 English', { 'Ret' => 0x75022ac4 } ], # pop/pop/ret in msvcp60.dll
```
Comment (adamziaja) [inline modules/exploits/windows/http/hp_power_manager_login.rb]: SP0-4

**Comment by OJ** [comment]
No description, repro, sample output? PR from "unknown repository"? Please see https://github.com/rapid7/metasploit-framework/blob/master/CONTRIBUTING.md

Rather than comment on your own code, why not update it so that the `SP0-SP4` details are added to the description?

**Comment by wvu-r7** [comment]
@adamziaja: Generally, it's not a good idea to delete your repo when you have a PR open. Could you please address @OJ's comments? Thanks! :)

**Comment by hmoore-r7** [comment]
Unless msvcp60.dll ships with the application, this isn't a reliable DLL to use

**Comment by wvu-r7** [comment]
Looks like the original ```msvcp60.dll``` p/p/r was removed. No clarification yet.

**Comment by OJ** [comment]
@adamziaja I'm going to close this PR because:

* It has no description.
* It has no sample output.
* It changes one of the `ret` values without justification.
* It has a modified target description that is incorrect.

If you feel like fixing these and trying again, then please do. Until then, closing.

**Pull request closed without merging**